pkg/sysfs/system.go

```diff
@@ -53,6 +54,16 @@ const (
 	DiscoverDefault DiscoveryFlag = (DiscoverCPUTopology | DiscoverMemTopology)
 )
 
+// MemoryType is an enum for the Node memory
+type MemoryType int
+
+const (
+	// MemoryTypeDRAM means that the node has regular DRAM-type memory
+	MemoryTypeDRAM MemoryType = iota
+	// MemoryTypePMEM means that the node has persistent memory
+	MemoryTypePMEM
```

Comment:
hbmem?

Reply:
So far only DRAM and PMEM were detected. I'll add HBMEM with the "alwasy-smaller-than-dram" assumption discussed below.